src/main/java/com/work/javaconsumer/service/PlayerConsumerService.java

```diff
@@ -34,6 +39,7 @@ public RegisterPlayerDTO registerPlayer(RegisterPlayerDTO registerPlayerDTO) {
     public WinDepositDTO deposit(WinDepositDTO winDepositDTO) {
         if (playerRepository.findById(winDepositDTO.getPlayerId()).isEmpty()) {
             System.out.println("No registered player with that id!");
```

Comment:
Also I am thinking - maybe you can add logs instead of System.out.println

Reply:
added log4j logger